src/components/Conference.tsx

```diff
@@ -235,6 +241,7 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
             const message = createDataChannelMessageSpeech(false);
             this.broadcastDataChannelMessage(message)
         })
+        this.setState({ audioMonitor } as IConferenceState);
```

Comment:
```ts
this.setState({ audioMonitor })
```
You can drop the `as <Type>` from `setState` calls since react.d.ts now uses better type information.

Reply:
Done